touchforms/backend/setup.py

```diff
@@ -15,6 +15,7 @@ def init_classpath():
 
     jardir = os.path.join(CUR_DIR, 'jrlib')
     jars = [k for k in os.listdir(jardir) if k.endswith('.jar')]
+    sys.path.append('/Users/benrudolph/.virtualenvs/commcare-hq/lib/python2.7/site-packages')
```

Comment:
I'll either have to direct this to the python_env in our code root, or start a new virtual env for touchforms and check it in to the repo. i think the later would be the least amount of configuration, but slightly clunky cause you'd have to check in the virtual env...